pytorch_lightning/core/lightning.py

```diff
@@ -390,7 +390,11 @@ def training_step(self, *args, **kwargs):
                 :paramref:`~pytorch_lightning.trainer.trainer.Trainer.truncated_bptt_steps` > 0.
 
         Return:
-            roch.Tensor or a dictionary with anything you want (must include the keyword 'loss')
+            Any of.
+
+            - `torch.Tensor`
```

Comment:
if only a tensor is returned, this is assumed to be the loss, right? Could you clarify this in the docstring?

Reply:
Done